ors-api/src/main/java/org/heigit/ors/api/requests/routing/RequestProfileParamsRestrictions.java

```diff
@@ -94,19 +91,18 @@ public class RequestProfileParamsRestrictions {
     @JsonIgnore
     private boolean hasWeight = false;
 
-    @Schema(name= PARAM_HAZMAT, description = "Specifies whether to use appropriate routing for delivering hazardous goods and avoiding water protected areas. Default is `false`. " +
-            "CUSTOM_KEYS:{'apiDefault':false,'validWhen':{'ref':'profile','value':['driving-hgv']}}",
+    @Schema(name= PARAM_HAZMAT, description = "Specifies whether to use appropriate routing for delivering hazardous goods and avoiding water protected areas. Default is `false`. ",
             extensions = { @Extension(name = "validWhen", properties = {
                     @ExtensionProperty(name = "ref", value = "profile"),
                     @ExtensionProperty(name = "value", value = "driving-hgv")}
-            )})
+            )},
+            defaultValue = "false")
```

Comment:
If `defaultValue = "false")` this is not always set, is the parameter included by default?

Reply:
This used if the profile is `driving-hgv`. But in that case it defaults to `false`.
What is the issue with this?

Reply:
Nothing. Was just a question